test/Feature/QuerySpec.hs

```diff
@@ -265,19 +265,8 @@ spec actualPgVersion = do
         [json|[{"id":1,"name":"Windows 7","clients":{"id":1,"name":"Microsoft"},"tasks":[{"id":1,"name":"Design w7"},{"id":2,"name":"Code w7"}]}]|]
         { matchHeaders = [matchContentTypeJson] }
 
-    it "requesting parent without specifying primary key" $
```

Comment:
This test has been moved to [EmbedDisambiguationSpec](https://github.com/PostgREST/postgrest/pull/1430/files#diff-6db2a928de7cd07d05747e73acfef7e8R144-R148)(which github hides from the default diff). It's shown to work with the FK rename.